### DISCUSSION TITLE
How to use git-credential-manager only for specific organization in Github

### DISCUSSION THREAD

**Discussion by aakashnand** [git-ecosystem/git-credential-manager#1652] [Q&A]
Hello git-credential-manager  community,

I recently started using git-credential-manager  to authenticate a repo which uses SSO in specific organization. 

When I installed git-credential-manager and logged in using browser for the first time, I saw that by default git-credential-manager  has access to all organizations in my account. Is there any way I can control this?

I have also read about this documentation mentioned on Github docs about restricting access of Oauth apps but I was not able control how git-credential-manager access my all organizations?

https://docs.github.com/en/organizations/managing-oauth-access-to-your-organizations-data/enabling-oauth-app-access-restrictions-for-your-organization

I also have found out that `git-credential-manager` is listed as one of the internal oauth app in following doc

https://docs.github.com/en/apps/oauth-apps/using-oauth-apps/internal-oauth-apps

But can control which organization I want to use git-credential-manager for?

**Comment by cassiasamp** [comment]
Any updates on this? 
I´m having the same issue and I can´t authorize the app for all other orgs in my account.

**Comment by hickford** [comment]
OAuth consent is at the user level.

But you can configure Git to use different usernames or helpers for different repositories or remote URLs. https://git-scm.com/docs/gitcredentials#_credential_contexts

**Reply by cassiasamp** [reply]
Hi @hickford, quick clarification..'at the user level' means that it will necessarily access all the orgs related to that user?
